spec/UserMiddleware.coffee

```diff
@@ -256,7 +256,7 @@ describe 'User Middleware', ->
         mock.restore()
       it 'should perform a token exchange and update user information', (done) ->
         action = 'application:url'
-        payload = "https://app.flowhub.io?code=#{code}"
+        payload = "https://app.flowhub.io?code=#{code}&state="
```

Comment:
Probably, though the main point here is that other query params than:

* `code` (for grant code on successful login)
* `error` (error message on failed login)

...should not matter